tests/core/test_broker.py

```diff
@@ -96,6 +99,16 @@ def test_pika_queues_from_args(
     assert pika_processor.queues == expected
 
 
+async def test_pika_raise_rasa_exception():
+    pika_broker = PikaEventBroker("host", "username", "password")
+    mock_channel = AsyncMock()
+    mock_channel.declare_queue = AsyncMock(side_effect=ChannelNotFoundEntity())
+    mock_exchange = AsyncMock()
+
+    with pytest.raises(RasaException):
+        await pika_broker._bind_queue("test_queue", mock_channel, mock_exchange)
```

Comment:
Is it possible to test this by calling a public method? It's always preferable to test from the public API of an object.

Reply:
I've edited the test after applying Tobias's suggestion to add the `ChannelNotFoundEntity` exception to `EventBroker `create method. Please check if everything looks ok.